test/testdata/resource-with-annotation.yaml

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
nit: test data doesn't have to be a pipeline definition. Even a simple Pod definition will be fine

Reply:
I thought of testing the actual pipeline, but it makes sense to use pod definitions for the sake of size and simplicity.